src/main/java/com/theelixrlabs/healthcare/constants/PatientConstants.java

```diff
@@ -0,0 +1,6 @@
+package com.theelixrlabs.healthcare.constants;
+
+public class PatientConstants {
+    public static final String CREATE_PATIENT_ENDPOINT = "/patients";
```

Comment:
Please give a proper endpoint for POST
which defines exact purpose of the API

Reply:
The document in which Shiju has shared, the endpoints were specified. So, I have used the same endpoints. So, should I change it or keep it the same?